data/korath/korath missions.txt

```diff
@@ -577,3 +577,19 @@ mission "Efreti: Distress Call 8"
 			`	You return to your ship and try plugging the chip into your systems again, which now displays two files: one in the Korath language, and one in your own. You open the file in your language, and it reads: "The <EDC-ship> is a ship of grief, bringing great pain in memories that sing for generations. Love is a void in the souls of many. Know all that <first> <last> is a true Efret by heart, if not by skin. The <EDC-ship> is theirs, to save lives and keep memories."`
 			`	You carefully store the chip in a safe place.`
 
+
+mission "Exile: Boarding Worldship Korath"
+	boarding
+	minor
+	destination "Earth"
+	source
+		government "Korath" "Korath (Civilian)"
+		category "Superheavy"
+		outfits "Afterburner (Stellar Class)"
+	on offer
+		conversation
+			`Docking with the gargantuan vessel allows you a front-row seat to witness the sheer scale of this ship. Your sensors detect tens of thousands of life-signs aboard, along with numerous other minor lifesigns, perhaps the trees that you can see in the massive dome atop the craft?`
```

Comment:
Well the minor life-signs are meant to be both fauna and the eggs of the Korath which are also in great number aboard the ship, so explaining it as only fauna is missing part of the context.

Reply:
Eggs would be best to add a specific mention of in that regard then, imo, especially to not get confused with animals present on the ship, including specifically at least aquaponic arrangements, last I knew.

Reply:
Adding for clarity:

The way it's worded without this suggestion, I had -no idea- the implication was meant to be eggs. To me, it always read as implying usage of livestock, so that's where I leaned in the suggestion.

Which is why I think if it's meant to be eggs (either only or as well), that needs to be changed or added in a different and more direct manner.